src/ExtEventLoop.php

```diff
@@ -178,8 +178,7 @@ public function tick()
 
         $this->futureTickQueue->tick();
 
-        // @-suppression: https://github.com/reactphp/react/pull/234#discussion-diff-7759616R226
-        @$this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
+        $this->eventBase->loop(EventBase::LOOP_ONCE | EventBase::LOOP_NONBLOCK);
```

Comment:
@jsor yes conceptually this test should do the trick. On a related note, I probably still have an old VM around at home on my NAS with an old development environment from back when I was actively contributing to reactphp (:cry:) so maybe I can try to dig it up and double check against it. I'll let you know ASAP!